packages/frontend/src/components/Editor/FlowStepWithAddButton.tsx

```diff
@@ -67,13 +37,16 @@ export default function FlowStepWithAddButton({
         isLastStep={isLastStep}
         isNested={isNested}
         // only allow reordering if there are more than 1 action steps
-        allowReorder={nonIfThenActionSteps.length > 1}
+        allowReorder={true}
```

Comment:
```suggestion
        allowReorder={allowReorder}
```

setting this to always be `true` will show the drag handle when there is nothing to drag. no real impact, just a display thing.

<img width="730" height="471" alt="Image" src="https://github.com/user-attachments/assets/1a5048aa-f387-4348-9cac-3706ff574db6" />